ff/serving/coordinator/coordinator.go

```diff
@@ -43,6 +44,28 @@ type Coordinator struct {
 	Spawner    JobSpawner
 }
 
+func (c *Coordinator) AwaitPendingSource(sourceNameVariant metadata.NameVariant, timeout int32) (*metadata.SourceVariant, error) {
```

Comment:
why not make the second variable a time.Duration so you don't have to add a comment making that clear?